### DISCUSSION TITLE
Device tenant?

### DISCUSSION THREAD

**Discussion by andymelichar** [bb-Ricardo/netbox-sync#50] [Q&A] [answered]
Can you clarify how devices are supposed to be assigned to a tenant?

**Comment by bb-Ricardo** [comment]
Well, good point. I doesn't exist. Would need to add a new config option right next to `vm_tenant_relation`.

Do you want to create an issue or should I?

**Reply by andymelichar** [reply]
https://github.com/bb-Ricardo/netbox-sync/issues/51

Thanks!!!

**Comment by bb-Ricardo** [comment, marked as answer]
This has been implemented
